README.md

```diff
@@ -91,87 +92,6 @@ For a module to act as the root module the only requirements are:
 
 ## 👩‍🍳 Recipes
 
-### Configuration with appConfig
```

Comment:
What is the logic you are following for what is a recipe vs an API?

Reply:
Recipe: A walkthrough or guided tutorial of topics
API Docs: A method-by-method description of the API surface. No walkthroughs.